.github/workflows/development.yml

```diff
@@ -0,0 +1,50 @@
+name: Development
```

Comment:
Yes, those should be required

Reply:
Do you have sufficient repo permissions to add settings for that? If not, can you open an IT support ticket to request elevated permissions, so that you can turn on branch protection rules?